<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="0" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-29T14:00:31 EDT" name="Default suite" finished-at="2024-03-29T14:00:32 EDT" duration-ms="405">
    <groups>
    </groups>
    <test started-at="2024-03-29T14:00:31 EDT" name="Default test" finished-at="2024-03-29T14:00:32 EDT" duration-ms="405">
      <class name="api.test.UserTests2">
        <test-method is-config="true" signature="setup()[pri:0, instance:api.test.UserTests2@626abbd0]" started-at="2024-03-29T14:00:31 EDT" name="setup" finished-at="2024-03-29T14:00:31 EDT" duration-ms="202" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testPostUser()[pri:1, instance:api.test.UserTests2@626abbd0]" started-at="2024-03-29T14:00:31 EDT" name="testPostUser" finished-at="2024-03-29T14:00:32 EDT" duration-ms="62" status="FAIL">
          <exception class="java.util.MissingResourceException">
            <message>
              <![CDATA[Can't find bundle for base name .\srcestesourcesoutes.properies, locale en_US]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.util.MissingResourceException: Can't find bundle for base name .\srcestesourcesoutes.properies, locale en_US
at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2062)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1700)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1603)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1558)
at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:864)
at api.endpoints.UserEndPoints2.getURL(UserEndPoints2.java:17)
at api.endpoints.UserEndPoints2.createUser(UserEndPoints2.java:23)
at api.test.UserTests2.testPostUser(UserTests2.java:34)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.util.MissingResourceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostUser -->
        <test-method signature="testGettUser()[pri:2, instance:api.test.UserTests2@626abbd0]" started-at="2024-03-29T14:00:32 EDT" name="testGettUser" finished-at="2024-03-29T14:00:32 EDT" duration-ms="4" status="FAIL">
          <exception class="java.util.MissingResourceException">
            <message>
              <![CDATA[Can't find bundle for base name .\srcestesourcesoutes.properies, locale en_US]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.util.MissingResourceException: Can't find bundle for base name .\srcestesourcesoutes.properies, locale en_US
at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2062)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1700)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1603)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1558)
at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:864)
at api.endpoints.UserEndPoints2.getURL(UserEndPoints2.java:17)
at api.endpoints.UserEndPoints2.readUser(UserEndPoints2.java:35)
at api.test.UserTests2.testGettUser(UserTests2.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.util.MissingResourceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGettUser -->
        <test-method signature="testUpdatetUser()[pri:3, instance:api.test.UserTests2@626abbd0]" started-at="2024-03-29T14:00:32 EDT" name="testUpdatetUser" finished-at="2024-03-29T14:00:32 EDT" duration-ms="4" status="FAIL">
          <exception class="java.util.MissingResourceException">
            <message>
              <![CDATA[Can't find bundle for base name .\srcestesourcesoutes.properies, locale en_US]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.util.MissingResourceException: Can't find bundle for base name .\srcestesourcesoutes.properies, locale en_US
at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2062)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1700)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1603)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1558)
at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:864)
at api.endpoints.UserEndPoints2.getURL(UserEndPoints2.java:17)
at api.endpoints.UserEndPoints2.updateUser(UserEndPoints2.java:45)
at api.test.UserTests2.testUpdatetUser(UserTests2.java:51)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.util.MissingResourceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdatetUser -->
        <test-method signature="testdeletetUser()[pri:4, instance:api.test.UserTests2@626abbd0]" started-at="2024-03-29T14:00:32 EDT" name="testdeletetUser" finished-at="2024-03-29T14:00:32 EDT" duration-ms="3" status="FAIL">
          <exception class="java.util.MissingResourceException">
            <message>
              <![CDATA[Can't find bundle for base name .\srcestesourcesoutes.properies, locale en_US]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.util.MissingResourceException: Can't find bundle for base name .\srcestesourcesoutes.properies, locale en_US
at java.base/java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:2062)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1700)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1603)
at java.base/java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1558)
at java.base/java.util.ResourceBundle.getBundle(ResourceBundle.java:864)
at api.endpoints.UserEndPoints2.getURL(UserEndPoints2.java:17)
at api.endpoints.UserEndPoints2.deleteUser(UserEndPoints2.java:58)
at api.test.UserTests2.testdeletetUser(UserTests2.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.util.MissingResourceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testdeletetUser -->
      </class> <!-- api.test.UserTests2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
